phpunit.xml.dist

```diff
@@ -22,14 +22,16 @@
             <directory>./</directory>
             <exclude>
                 <directory>./Tests/</directory>
+                <directory>./DataFixtures/</directory>
                 <directory>./Resources/</directory>
-                <directory>./DependencyInjection/</directory>
                 <directory>./vendor/</directory>
-
-                <!-- Ignore ODM driver -->
-                <directory>./Document/</directory>
+                <directory>./coverage/</directory>
```

Comment:
Where does this path come from?

Reply:
IMO it could also be removed, but I am trying to unify to use the same file on every project, so I copied it from SonataAdminBundle: https://github.com/sonata-project/SonataAdminBundle/blob/3.x/phpunit.xml.dist

My intention is, once all projects got the same file (only this one is missing AFAIK) do a PR on dev-kit to change all of them at once, because they can be all equal

Reply:
We don't have the path in our bundles. So we could remove this on all bundles (via dev-kit).

Reply:
yes, that path is only used when generating coverage, maybe for that reason is excluded. We should need to test, but I think it is not necessary at all :)